src/releasers/php.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ReleasePR, ReleaseCandidate, PackageName} from '../release-pr';
+import {Update, VersionsMap} from '../updaters/update';
+
+// Generic
+import {Changelog} from '../updaters/changelog';
+// PHP Specific.
+import {RootComposerUpdatePackage} from '../updaters/root-composer-update-package';
+import {ReleasePRConstructorOptions} from '..';
+
+const CHANGELOG_SECTIONS = [
+  {type: 'feat', section: 'Features'},
+  {type: 'fix', section: 'Bug Fixes'},
+  {type: 'perf', section: 'Performance Improvements'},
+  {type: 'revert', section: 'Reverts'},
+  {type: 'docs', section: 'Documentation'},
```

Comment:
Non blocking, do you think that folks in the PHP community will want to create releases when they create doc updates?

For the Node.js releaser we opt to only release on `fix`, `feat`, and breaking changes.

Reply:
Yes, I need to add hidden: true